src/main/scala/chisel3/util/BitPat.scala

```diff
@@ -77,6 +77,28 @@ object BitPat {
     val len = if (x.isWidthKnown) x.getWidth else 0
     apply("b" + x.litValue.toString(2).reverse.padTo(len, "0").reverse.mkString)
   }
+
+  implicit class fromUIntToBitPatComparable(x: UInt) extends SourceInfoDoc {
```

Comment:
Does this import correctly? eg, if you import `chisel3._`, does it automatically give you the BitPat conversions? Or maybe since this is in chisel3.util, perhaps it should work if you `import chisel3.util._`? But this seems to imply that one needs to import `chisel3.util.BitPat._`

We should also make sure we have a uniform way of defining implicit conversions, either in their object (as in this case) or in the package (in everywhere else in chisel)

Reply:
This does appear to import correctly (I copied an existing pattern), but we should discuss this at the dev meeting.